hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/Ls.java

```diff
@@ -245,9 +263,27 @@ protected void processPath(PathData item) throws IOException {
       return;
     }
     FileStatus stat = item.stat;
+
```

Comment:
this is cherry picking in and then commenting out something which seems unrelated.

Reply:
This was because I involved changes in hadoop-common between trunk and branch-2 due the reason above. I have redone it and it should be excluded now.